roxctl/central/generate/generate.go

```diff
@@ -14,14 +13,14 @@ import (
 	"github.com/stackrox/rox/pkg/buildinfo"
 	"github.com/stackrox/rox/pkg/certgen"
 	"github.com/stackrox/rox/pkg/env"
-	"github.com/stackrox/rox/pkg/errox"
 	"github.com/stackrox/rox/pkg/features"
 	"github.com/stackrox/rox/pkg/images/defaults"
 	"github.com/stackrox/rox/pkg/mtls"
 	"github.com/stackrox/rox/pkg/renderer"
 	"github.com/stackrox/rox/pkg/roxctl"
 	"github.com/stackrox/rox/pkg/utils"
 	"github.com/stackrox/rox/pkg/zip"
+	"github.com/stackrox/rox/roxctl/common"
```

Comment:
What about `roxctl "github.com/stackrox/rox/roxctl/common/errors"`?

Reply:
@0x656b694d I have thought about it. I think it does not bring us much. When you read it, it looks good as long we are prefixing errors with `Err` - i.e. `common.ErrSomething`. Adding additional namespace level, will not improve much and it will complicate imports because we will have a collision with the default `errors` package.